common/lib/container-definitions/src/error.ts

```diff
@@ -36,14 +38,30 @@ export enum ContainerErrorType {
 /**
  * Base interface for all errors and warnings at container level
  */
-export interface IErrorBase {
+export interface IErrorBase extends Partial<Error> {
```

Comment:
What's the value of Partial<Error> to consumers? It feels none, and the only benefit is ensuring that types for message/name/stack are compatible with Error definitions, right? I.e. it's more of an internal minimal help.
Maybe other way to ask - can it be required (non-partial)?

Reply:
The breaking change validation stuff blew up when I made it required.  I would like it to be required, but felt unsure of the ramifications of doing so on account of that, so thought it was reasonable to start here.

Reply:
I can revisit making it required in the next release, when I have time to think through the N/N-1 concerns